app/models/post_image_manager.rb

```diff
@@ -27,7 +29,17 @@ def add_file(file)
   end
 
   ##
-  # Clears the manager of all currently exisiting image uploaders and delete's their cache directories
+  # Adds an image that was downloaded from the SSE website repo
+  #
+  # Params:
+  # +file+:: A PostImage object representing the downloaded image
```

Comment:
Can you change the name of the param in the method in the comment to match the actual parameter in this method?

Reply:
Fixed